flax/linen/stochastic.py

```diff
@@ -27,6 +27,12 @@
 class Dropout(Module):
   """Create a dropout layer.
 
+    Note when using `:meth:`Module.apply() <flax.linen.Module.apply>` remember
+    to include an rng seed named 'dropout'. This would look something like:
```

Comment:
Suggestion: minor refactoring

```suggestion
    Note: When using `:meth:`Module.apply() <flax.linen.Module.apply>`, make sure
    to include an RNG seed named `'dropout'`. For example:
```